doc/SPO-MANUAL-MIGRATION-GUIDE.md

```diff
@@ -0,0 +1,564 @@
+echo "" >> $BACKUP_DIR/backup-summary.txt
+echo "Profile bindings backed up:" >> $BACKUP_DIR/backup-summary.txt
+find $BACKUP_DIR/bindings-recordings -name "*.json" -exec basename {} \; | sort >> $BACKUP_DIR/backup-summary.txt
+
+cat $BACKUP_DIR/backup-summary.txt
+```
+
+**Actual output:**
+```
+=== Backup Summary ===
+Total files backed up: 6
+
+SPOD configuration: Yes
+
+Profiles backed up:
+rawselinuxprofiles-openshift-security-profiles-errorlogger.json
+seccompprofiles-openshift-security-profiles-profile-block-all.json
+selinuxprofiles-openshift-security-profiles-errorlogger-selinuxd-test.json
+
+Profile bindings backed up:
+profilebindings-openshift-security-profiles-profile-binding-seccomp.json
+profilebindings-openshift-security-profiles-profile-binding-selinuxd.json
+```
+
+## Step 4: Delete SPO Objects in Order
+
+**IMPORTANT**: Delete profile bindings first to prevent the operator from automatically reapplying profiles to workloads.
+
+### Delete Profile Bindings First
+```bash
+# List and delete all profile bindings
+oc get profilebindings -n openshift-security-profiles
+oc delete profilebindings --all -n openshift-security-profiles
+```
+
+**Actual command and output:**
+```bash
+oc delete profilebinding profile-binding-seccomp profile-binding-selinuxd -n openshift-security-profiles
+```
+```
+profilebinding.security-profiles-operator.x-k8s.io "profile-binding-seccomp" deleted
+profilebinding.security-profiles-operator.x-k8s.io "profile-binding-selinuxd" deleted
+```
+
+### Update Workloads to Remove Profile References
+
+Next, update all workloads to remove references to SPO profiles. You must either recreate pods without these profiles, or update deployments, statefulset, and other workloads to remove the profile references from their pod templates.
+
+
+### Delete Security Profiles
+```bash
+# Delete all security profiles
+oc delete seccompprofiles,selinuxprofiles,rawselinuxprofiles --all -n openshift-security-profiles
+```
+
+**Example output:**
+```
+seccompprofile.security-profiles-operator.x-k8s.io "profile-block-all" deleted
+selinuxprofile.security-profiles-operator.x-k8s.io "errorlogger-selinuxd-test" deleted
+rawselinuxprofile.security-profiles-operator.x-k8s.io "errorlogger" deleted
+```
+
+### Wait for Profile Cleanup
+```bash
+# Verify all profiles are deleted
+oc get seccompprofiles,selinuxprofiles,rawselinuxprofiles -A
+```
+
+## Step 5: Uninstall SPO Operator
+
+### Delete SPO Daemon
+```bash
+# Delete the spod resource before removing the operator
+oc delete spod spod -n openshift-security-profiles
+```
+
+### Delete Operator
+
+To uninstall the Security Profiles Operator (SPO) Operator, follow these steps:
+
+### Uninstall via OperatorHub (Recommended)
+1. Open the OpenShift Console.
+2. Go to **Operators** → **Installed Operators**.
+3. Select **Security Profiles Operator** in the `openshift-security-profiles` namespace.
+4. Click **Uninstall**.
+5. Confirm the uninstallation.
+
+
+### Delete Namespace
+```bash
```

Comment:
Before delete the namespace, you also need to run below command:
`oc delete mutatingwebhookconfigurations spo-mutating-webhook-configuration`